pylib/cqlshlib/tracing.py

```diff
@@ -47,7 +55,7 @@ def print_trace(shell, trace):
     names = ['activity', 'timestamp', 'source', 'source_elapsed', 'client']
 
     formatted_names = list(map(shell.myformat_colname, names))
-    formatted_values = [list(map(shell.myformat_value, row)) for row in rows]
+    formatted_values = [[format_trace(row[0])] + list(map(shell.myformat_value, row[1:])) for row in rows]
```

Comment:
Originally I was wondering if there wasn't a way to do this in make_trace_rows() when the distinction between the first element and the others is more explicit.  But since it's very definitely the formatting function that's swapping out newlines we're somewhat limited on options... so I think this is probably the best available approach.